src/software/field_tests/movement_robot_field_test.py

```diff
@@ -204,17 +209,18 @@ def test_one_robots_square(field_test_runner):
     )
     tactics = [tactic_0, tactic_1, tactic_2, tactic_3]
 
-    for tactic in tactics:
-        print(f"Going to {tactic.destination}")
-        params = AssignedTacticPlayControlParams()
-        params.assigned_tactics[id].move.CopyFrom(tactic)
-
-        field_test_runner.set_tactics(params, True)
-        field_test_runner.run_test(
-            always_validation_sequence_set=[[]],
-            eventually_validation_sequence_set=[[]],
-            test_timeout_s=4,
-        )
+    for i in range(3):
+        for tactic in tactics:
+            print(f"Going to {tactic.destination}")
+            params = AssignedTacticPlayControlParams()
+            params.assigned_tactics[id].move.CopyFrom(tactic)
+
+            field_test_runner.set_tactics(params, True)
+            field_test_runner.run_test(
+                always_validation_sequence_set=[[]],
+                eventually_validation_sequence_set=[[]],
+                test_timeout_s=4,
+            )
```

Comment:
you probably should revert the changes in this file. I don't think you meant to commit these.

Reply:
Yeah, lets keep the changes to `test_basic_rotation`, but we can revert the rest. 

I've also realized that when we use `pytest_main` to run a test, we can pass in the following arg: `--test_filter <test_function_name>` to filter which tests we will be running. So at some point we can comment back in the other tests in this file.